tests/system/libraries/ChromeLib.py

```diff
@@ -122,6 +122,7 @@ def _waitForStartMarker(self, spy, lastSpeechIndex):
 			actualSpeech = spy.get_speech_at_index_until_now(lastSpeechIndex)
 			if self._wasStartMarkerSpoken(actualSpeech):
 				break
+			spy.emulateKeyPress('F6')

```

Comment:
why was this added?

Reply:
When running locally, and chrome is already open (windowed) the test does not open in kiosk mode. For some reason focus isn't always on the document in that case. I will remove this.

It is a little strange that we have a loop that doesn't change anything between tests now.